src/components/SideBar.js

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+
+const Bar = styled.div`
+  position: relative;
+  display: flex;
+  z-index: 2;
+  flex-direction: column;
+  height: 100%;
+  border-right: 2px solid;
+  border-radius: 0;
+  border-color: var(--caret-color);
+  background-color: var(--color-historyPage);
+  transition: 0.8s ease;
+`;
+
+const Icon = styled.img`
+  position: absolute;
+  outline: none;
+  z-index: 2;
+  width: 50px;
+  height: 80px;
+  border-left: 0;
```

Comment:
저는 아래와 같은 순서로 작성하는 편인데, 이 부분에 대해서는 잠시 후에 이야기 나눠보고 결정하면 좋을 것 같아요!

```suggestion
  position: absolute;
  z-index: 2;
  width: 50px;
  height: 80px;
  border-left: 0;
  outline: none;
```